File: android/app/src/main/java/com/example/fugliara/ccs002/server/dataobjects/UserFacebookItem.java
package com.example.fugliara.ccs002.server.dataobjects;

public class UserFacebookItem {

    @com.google.gson.annotations.SerializedName("id")
    public String id;
    @com.google.gson.annotations.SerializedName("email")
    public String email;
    @com.google.gson.annotations.SerializedName("name")
    public String name;
    @com.google.gson.annotations.SerializedName("dateTime")
    public String date;
    @com.google.gson.annotations.SerializedName("token")
    public String token;

    public UserFacebookItem() {
        id = "";
        date = "";
        name = "";
        email = "";
        token = "";
    }
}

File: azure/transporterSolution/transporterService/DataObjects/UserFacebookItem.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;
using transporterService.DataTranspObjects;

namespace transporterService.DataObjects
{
    public partial class UserFacebookItem : BaseItem
    {
        //public virtual UserItem UserItem { get; set; }

        public string DateTime { get; set; }
        public string Token { get; set; }
        public string Email { get; set; }
        public string Name { get; set; }
    }
}